scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -118,6 +116,41 @@ public static BuildTargetResult PublishDebFileToDebianRepo(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        [Environment("DOCKER_HUB_REPO")]
+        [Environment("DOCKER_HUB_TRIGGER_TOKEN")]
+        public static BuildTargetResult TriggerDockerHubBuilds(BuildTargetContext c)
+        {
+            string dockerHubRepo = Environment.GetEnvironmentVariable("DOCKER_HUB_REPO");
+            string dockerHubTriggerToken = Environment.GetEnvironmentVariable("DOCKER_HUB_TRIGGER_TOKEN");
+
+            Uri baseDockerHubUri = new Uri("https://registry.hub.docker.com/u/");
+            Uri dockerHubTriggerUri;
+            if (!Uri.TryCreate(baseDockerHubUri, string.Format("{0}/trigger/{1}/", dockerHubRepo, dockerHubTriggerToken), out dockerHubTriggerUri))
```

Comment:
Consider using the $ interpolated string syntax instead of string.Format to be consistent with the rest of the file.  IMO the $ syntax is more readable.